bindings/pyroot/pythonizations/src/RTensorPyz.cxx

```diff
@@ -20,6 +20,7 @@
 
 ////////////////////////////////////////////////////////////////////////////
 /// \brief Adopt memory of a Python object with array interface using an RTensor
+/// \param[in] self self object
```

Comment:
This is also always null, so we can write `self Always null, since this is a module function.`.

Reply:
Fixed.